torch/csrc/jit/mobile/import.cpp

```diff
@@ -151,6 +153,29 @@ void parseMethods(
         expect_field(table, "register_size", BYTECODE_INDEX_REGISTER_SIZE)
             .toInt();
 
+    // If there exist a tuple in consts_list with this format
+    // ('tensor_jit_index', 4) fetch the tensor at index 4 from tensor table
+    // generated in jit, and push it to the new constant list
+    // *updated_constant_vals*.
+    std::vector<IValue> updated_constant_vals;
+    for (const auto& const_item : consts_list) {
+      if (const_item.isTuple()) {
```

Comment:
Is it possible that a constant may exactly look like a tuple? If so can we add some string identifier that makes it unlikely that such a tuple is actually a constant we want?